test/test_quantization.py

```diff
@@ -573,10 +573,13 @@ def checkQuantized(model):
         checkQuantized(model)
 
     @unittest.skip("temporarily disable the test")
-    @given(qengine=st.sampled_from(("qnnpack", "fbgemm")))
+    @given(qengine=st.sampled_from(("fbgemm",)))
     def test_quantized_rnn(self, qengine):
         d_in, d_hid = 2, 2
 
+        # TODO: qlinear_prepack_fp16 currently doesn't support QNNPACK
+        # re-add "qnnpack" to the engine set when this is supported
+
```

Comment:
This change actually revealed a bug: before we would be blindly packing in the FBGEMM format regardless of the engine setting